src/pages/non-pass-finalisation/cat-a-mod1/non-pass-finalisation.cat-a-mod1.page.ts

```diff
@@ -91,7 +91,7 @@ export class NonPassFinalisationCatAMod1Page extends BasePageComponent implement
   ) {
     super(platform, navController, authenticationProvider);
     this.form = new FormGroup({});
-    this.activityCodeOptions = activityCodeModelList;
```

Comment:
is there a need for changing this for a category a DLG examiner won't complete? same for B/MOD2/ADI

Reply:
yes, yes it it

Reply:
so you cant just use what was there before? i.e.

this.activityCodeOptions = activityCodeModelList;

Reply:
actually no.....it doesnt need to be the same. if i leave it as before it will default to true and add the additional values. So as such i need to set it to false.

Reply:
same for all none required categories, ie B, HOME.

Reply:
are you not able to change the default to no then?

Reply:
no, cant change the default to false.  If i do then i need to change every office page and the test-selector and if i change the test selector then im back where i was friday afternoon with the value no patching correctly.  This may be solved with having two completely seperate functions withing activity-codes constants and two different selectors.